lib/rules/no-restricted-imports.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @fileoverview Restrict usage of specified node imports.
+ * @author Guy Ellis
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+    var restrictedImports = context.options;
+
+    // if no imports are restricted we don"t need to check
+    if (restrictedImports.length === 0) {
+        return {};
+    }
+
+    return {
+        "ImportDeclaration": function(node) {
+            if (node && node.source && node.source.value) {
+
+                var value = node.source.value.trim();
+
+                if (restrictedImports.indexOf(value) !== -1) {
```

Comment:
@nzakas I just added this test: https://github.com/guyellis/eslint/blob/no-restricted-imports/tests/lib/rules/no-restricted-imports.js#L24

`restrictedImports` is an array I believe. So that syntax should work. Does that look good?